package.json

```diff
@@ -3226,13 +3326,12 @@
         "reset": "npm run clean && npm install --no-save",
         "watch": "tsc -watch -m commonjs -p ./",
         "watch-ui": "pushd \"./src/ui\" && npm run watch --no-save && popd",
-
         "postinstall": "node ./node_modules/vscode/bin/install && pushd \"./src/ui\" && npm install --no-save && popd",
         "prepush": "npm run build",
         "vscode:prepublish": "npm run reset && npm run bundle"
     },
     "dependencies": {
-        "copy-paste": "1.3.0",
+        "clipboardy": "^1.2.3",
```

Comment:
Please remove the `^` -- I like to have the dependencies locked down for reproducible builds